GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -689,7 +689,9 @@ private void writeAnswer(
         int col = startColumn;
         for (String cellValue : cells) {
             if (questionType == QuestionType.NUMBER) {
-                createCell(row, col, cellValue, mNumberStyle, Cell.CELL_TYPE_NUMERIC);
```

Comment:
Can you also please clear out the `mNumberStyle` variable earlier since its commented out?